src/main/java/com/ankitmahapatra/dofirewallupdater/DigitalOceanFirewallUpdaterApplication.java

```diff
@@ -7,4 +7,20 @@ public static void main(String[] args){
     			new DigitalOceanFirewallUpdaterApplication();
     	System.out.println(" Starting with firewall update ...");
     }
+    
+    private String getMyPublicIPAddress() throws UnknownHostException
+        {
+            String host = "https://api.ipify.org/";
+
+            try {
+                InetAddress ip = InetAddress.getByName(new URL(host)
```

Comment:
I would need my public address to be returned by the method and not of api.ipfy.org . We need to use the server provided by IPIFY to get the caller's public IP address.

Reply:
Okay, the expectation here is that using the APIs of IPIFY, your public IP address has to be retrieved. Is my understanding correct?

Reply:
Yes, you might want to remove the `throws exception` as well since we already have a try-catch block.

Reply:
Sure, made the necessary changes. Thanks for the feedback.